cmd/cmd.go

```diff
@@ -85,6 +85,17 @@ func runScan(cmd *cobra.Command, args []string) {
 		logger.Fatal("invalid leak-exit-code: error=%q", err.Error())
 	}
 
+	gitleaksConfig, err := cmd.Flags().GetString("gitleaks-config")
+	if err != nil {
+		logger.Fatal("invalid gitleaks-config: error=%q", err.Error())
+	}
+
+	// Providing a gitleaks-config via command line arguments takes
+	// precedence over gitleaks config set in the leaktk config file
+	if len(gitleaksConfig) != 0 {
+		cfg.Scanner.Patterns.Gitleaks.ConfigPath = gitleaksConfig
```

Comment:
With a custom config passed in, we probably want to set auto fetch to false and disable the expiration check 👌

Just to keep it from overwriting the passed in file or freaking out of it has an old timestamp.